index.js

```diff
@@ -28,13 +29,25 @@ const chooseKey = key =>
 const decode = partialRight(jwt.decode, [{ complete: true }])
 
 const enforce = token =>
-  token || Promise.reject(Boom.unauthorized('null token not allowed'))
+  token || reject(Boom.unauthorized('null token not allowed'))
+
+const forbidden = err =>
+  reject(Boom.forbidden(err))
+
+const isIssWhitelistError =
+  pathEq(['name'], 'IssWhitelistError')
```

Comment:
🐏 ⛳️

Could we use ramda's `is` for this? i.e., `is(IssWhitelistError)`?

https://ramdajs.com/docs/#is

Reply:
🏌️‍♂️
Yes - that worked perfectly! Much better